# iffe determines wrongly the vmalloc libraries.
# for icc _lib_mallopt, _lib_mallinfo, and _lib_mstats should be 0 in /FEATURE/vmalloc
# inspired by http://gnats.netbsd.org/43870
--- lib/vmalloc/features/vmalloc.orig   2014-04-13 22:40:25.000000000 +0200
+++ lib/vmalloc/features/vmalloc        2014-09-08 13:25:40.230441395 +0200
@@ -13,9 +13,6 @@
 sys stat
 typ ssize_t
 hdr malloc
-lib mallopt
-lib mallinfo
-lib mstats
 hdr alloca
 hdr dlfcn
